glusterd2/xlator/validation.go

```diff
@@ -19,6 +19,30 @@ func validateReplica(v *volume.Volinfo, key string, value string) error {
 	return nil
 }
 
+func validateBitrot(v *volume.Volinfo, key string, value string) error {
+	switch key {
+	case "scrub-throttle":
+		acceptedThrottleValues := []string{"lazy", "normal", "aggressive"}
+		if Contains(value, acceptedThrottleValues) {
+			return nil
+		}
+		return errors.New("Invalid option value")
```

Comment:
Suggestion: it'll be nicer for the errors to contain some more detail. w.r.t to which key is invalid and which values are valid ones. You can propagate the same back to  CLI too.

```go
var err error
...
	err := fmt.Errorf("Invalid value specified for option '%s'. Possible values: %s", key, strings.Join(acceptedThrottleValues, ", "))
...
return err
```

Reply:
Done